docs/PROVISIONING_AWS.adoc

```diff
@@ -0,0 +1,142 @@
+= OpenShift on AWS EC2 using CASL
+
+== Local Setup (one time, only)
+
+NOTE: These steps are a canned set of steps serving as an example, and may be different in your environment.
+
+Before getting started following this guide, you'll need the following:
+
+* Access to an AWS account with the proper policies to create resources (see details below)
+* Docker installed
+  * RHEL/CentOS: `yum install -y docker`
+  * Fedora: `dnf install -y docker`
+  * **NOTE:** If you plan to run docker as yourself (non-root), your username must be added to the `docker` user group.
+* Ansible 2.5 or later installed
+  * link:https://docs.ansible.com/ansible/latest/installation_guide/intro_installation.html[See Installation Guide]
+[source,bash]
+----
+cd ~/src/
+git clone https://github.com/redhat-cop/casl-ansible.git
+----
+
+* Run `ansible-galaxy` to pull in the necessary requirements for the CASL provisioning of OpenShift on AWS:
+
+NOTE: The target directory ( `galaxy` ) is **important** as the playbooks know to source roles and playbooks from that location.
+
+[source,bash]
+----
+cd ~/src/casl-ansible
+ansible-galaxy install -r casl-requirements.yml -p galaxy
+----
+
+== AWS Setup
+
+The following needs to be configured in your AWS account.
+
+* An link:https://console.aws.amazon.com/iam/home?#/users[IAM User] must be created with API access, and the ability to provision VPCs, Subnets, Instances, EBS Volumes, etc. (see note below on policy specifics)
+  * Once created, set export `AWS_ACCESS_KEY_ID` and `AWS_SECRET_ACCESS_KEY` to your environment.
+* Create a link:http://docs.aws.amazon.com/AWSEC2/latest/UserGuide/ec2-key-pairs.html#having-ec2-create-your-key-pair[Key-pair in AWS]
+* Create a link:https://console.aws.amazon.com/route53/home?#hosted-zones:[Hosted Zone] in your AWS Account.
+
+=== Example Policy Assignment
```

Comment:
I think this is "muddying" the waters quite a bit here, and I think it's very specific to a specific user (or just a few users). I'd prefer that we don't include as much as it is here within these docs, but maybe link out to another doc if/when this section is needed?